server/controllers/admin.py

```diff
@@ -123,3 +129,55 @@ def assignment(cid, aid):
     return render_template('staff/course/assignment.html', assignment=assgn,
                            form=form, courses=courses,
                            current_course=current_course)
+
+@admin.route("/course/<int:cid>/enrollment",
+             methods=['GET', 'POST'], defaults={'page': 1})
+@admin.route("/course/<int:cid>/enrollment/page/<int:page>",)
```

Comment:
`page` should be a query param since it's the same template - `/courses/1/enrollment/?page=5`